api/resolvers/wallet.js

```diff
@@ -551,7 +551,7 @@ const resolvers = {
 
       const logger = walletLogger({ wallet, models })
       await models.wallet.delete({ where: { userId: me.id, id: Number(id) } })
-      logger.info('wallet detached')
+      logger.info('details for receiving deleted')
```

Comment:
If this wallet is using device sync, isn't it possible there were no receive creds ever saved or deleted?